docs/source/execute_api.rst

```diff
@@ -9,72 +9,109 @@ nbconvert provides a convenient way to execute the input cells of an
 as a .ipynb file.
 
 In this section we show how to execute a ``.ipynb`` notebook
-document saving the result in notebook format. If you need to export
+document and save the result in notebook format. If you need to export
 notebooks to other formats, such as reStructured Text or Markdown (optionally
 executing them) see section :doc:`nbconvert_library`.
 
-Executing notebooks can be very helpful, for example, to run all notebooks
-in Python library in one step, or as a way to automate the data analysis in
-projects involving more than one notebook.
+Executing notebooks can be helpful functionality for some use cases.
+For example, nbconvert can run all notebooks in a folder in one step.
+Automating the data analysis in projects involving more than one notebook
+is also possible.
+
+Notebook execution can be done either from the command line or programmatically
```

Comment:
I think we should add a note that nbclient is the natural successor for programmatic execution of notebooks, but that the interface remains here in nbconvert for extension and mixin of the preprocessor concept, and that papermill has a more extensive and robust command line interface than nbconvert for stand alone notebook execution. Many people using nbconvert execute don't know about those other options or when one is better to use.